qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -49,57 +55,76 @@ Required fields for centralized Qiita
 system:
 
-+-------------------+-------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Field name        | Format                        | Description                                                                                                                                                                                          |
-+===================+===============================+======================================================================================================================================================================================================+
-| ``sample_name``   | free text with restrictions   | Identifies a sample. It is the primary key, must be unique and should match the ones in the sample template. Allowed characters are alphabetic ``[A-Za-z]``, numeric ``[0-9]``, and periods ``.``.   |
-+-------------------+-------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++-------------------+-------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Field name        | Format                        | Description                                                                                                                                                                                                                                                   |
++===================+===============================+===============================================================================================================================================================================================================================================================+
+| ``sample_name``   | free text with restrictions   | Identifies a sample. It is the primary key, must be unique and should match the ones in the sample information file. Allowed characters are alphabetic ``[A-Za-z]``, numeric ``[0-9]``, and periods ``.``. Must match the sample_name in the sample template. |
++-------------------+-------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
 Required fields for EBI submission
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 These are the columns required for successfully submit your data to EBI:
```

Comment:
Just noticed this, not part of this PR but this should be changed to:

`These are teh columns required to successfully submit your data to EBI`